csharp/test/Ice/namespacemd/msbuild/client/client.csproj

```diff
@@ -8,6 +8,7 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <OutputPath>.</OutputPath>
+    <Configurations>Release;Debug</Configurations>
```

Comment:
Why is this required?

Reply:
This was added by VS Studio when I removed the interceptor test.